tools/build_script_generator/cmake/module.lb

```diff
@@ -35,8 +35,8 @@ def prepare(module, options):
 def build(env):
     project_name = env[":build:project.name"]
     build_path = env[":build:build.path"]
-    env.append_metadata_unique("elf.release", join(build_path, "cmake-build-release", project_name + ".elf"))
-    env.append_metadata_unique("elf.debug",   join(build_path, "cmake-build-debug",   project_name + ".elf"))
+    env.append_metadata_unique("elf.release", join(build_path, "cmake-build-release", project_name + ".elf").replace('\\','/'))
+    env.append_metadata_unique("elf.debug",   join(build_path, "cmake-build-debug",   project_name + ".elf").replace('\\','/'))
```

Comment:
I'd rather make this a part of the `openocd.cfg.in`/`gdbinit` template code, with a jinja2 filter (maybe `windowsify`), because I'm pretty sure I'll forget this `.replace` when adding new paths or refactoring this module.

Inside the `module.lb`:
```py
def windowsify(path):
    return path.replace("\\", "/")

env.template(..., filter={"windowsify": windowsify})
```

Then inside the template:
```
{{ path | windowsify }}
```